Source/AwsCommonRuntimeKit/http/HTTPRequestOptions.swift

```diff
@@ -3,22 +3,26 @@
 import Foundation
 
 public struct HTTPRequestOptions {
-    public typealias OnIncomingHeaders = (_ stream: HTTPStream,
-                                          _ headerBlock: HTTPHeaderBlock,
+
+    public typealias OnInterimResponse = (_ statusCode: Int32,
                                           _ headers: [HTTPHeader]) -> Void
-    public typealias OnIncomingHeadersBlockDone = (_ stream: HTTPStream,
-                                                   _ headerBlock: HTTPHeaderBlock) -> Void
-    public typealias OnIncomingBody = (_ stream: HTTPStream, _ bodyChunk: Data) -> Void
-    public typealias OnStreamComplete = (_ stream: HTTPStream, _ error: CRTError?) -> Void
+    public typealias OnResponse = (_ statusCode: Int32,
+                                   _ headers: [HTTPHeader]) -> Void
+    public typealias OnTrailer = (_ headers: [HTTPHeader]) -> Void
+    public typealias OnIncomingBody = (_ bodyChunk: Data) -> Void
```

Comment:
utterly trivial: declare OnIncomingBody before OnTrailer, since that's the body arrives before the trailer

Reply:
/// docs

Reply:
Thanks, fixed.